marabunta/model.py

```diff
@@ -30,8 +30,8 @@ def versions(self):
 class MigrationOption(object):
 
     def __init__(self, install_command=None, install_args=None):
-        self.install_command = install_command or u'odoo'
-        self.install_args = install_args or u''
+        self.install_command = install_command or 'odoo'
+        self.install_args = install_args or ''
```

Comment:
Would be better to keep the `u'...'` for retro compatibility.

Otherwise you need to do:

`from __future__ import unicode_literals`

http://python-future.org/compatible_idioms.html#unicode-text-string-literals